### PR TITLE
BXMSDOC-7814 Additional 7.12 changes.

### PR DESCRIPTION
**Pull request by briandooley**
Jira: [BXMSDOC-7814](https://issues.redhat.com/browse/BXMSDOC-7814)
RHPAM Rendering: http://file.ork.redhat.com/~bdooley/BXMSDOC-7814-RHPAM/#clustering-datagrid-proc_clustering-dev
RHDM Rendering: http://file.ork.redhat.com/~bdooley/BXMSDOC-7814-RHDM/#clustering-datagrid-proc_clustering-dev

### CODE REVIEW

**Review by tomasdavidorg — APPROVED** [review]
Looks good to me. Thanks @briandooley 🙂

File: doc-content/enterprise-only/installation/clustering-bc-configure-standalone_proc.adoc [hunk 1]
```diff
@@ -48,6 +48,9 @@ mount -o rw,sync,actimeo=1 <NFS_SERVER_IP>:<DATA_SHARE>/kie-wb-playground kie-wb
 * `<AMQ_USER>` and `<AMQ_PASSWORD>` are the credentials that you defined when creating the AMQ Broker.
 * `<AMQ_BROKER_IP_ADDRESS>` is the IP address of the AMQ Broker.
 * `<DATA_GRID_NODE_IP>` is the IP address where {DATAGRID} is installed.
+* `<SERVER_NAME>` is the server name specified in your {DATAGRID} server configuration.
+* `<SASL_QOP>` is the combination of auth, auth-int and auth-conf values for your {DATAGRID} server configuration.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-bc-configure-standalone_proc.adoc]: auth, auth-int and auth-conf

These should be in code, right?

File: doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc [hunk 2]
```diff
@@ -18,11 +18,54 @@ NOTE: Do not install {DATAGRID} on the same node as {CENTRAL}.
...
-. To run {DATAGRID}, navigate to `__JDG_HOME__/bin` and enter the following command:
+. To add a {DATAGRID} user, navigate to `__JDG_HOME__/bin` and enter the following command:
 +
 [source]
 ----
-$ ./standalone.sh -c clustered.xml -b <HOST>
+$ ./cli.sh user create <DATAGRID_USER_NAME> -p <DATA_GRID_PASSWORD> -r default
+----
++
+. To change Data Grid server memory parameters, open `__JDG_HOME__/bin/server.conf` and replace
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc]: Why not  use the Data Grid variable here?
open the `__JDG_HOME__/bin/server.conf`file.

I really don't like splitting code examples like this. I would rewrite:

...open the `__JDG_HOME__/bin/server.conf`file and locate the following line:
-Xms64m -Xmx512m -XX:MetaspaceSize=64M

. Replace this line with the following values:
-Xms256m -Xmx2048m -XX:MetaspaceSize=256M

File: doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc [hunk 3]
```diff
@@ -18,11 +18,54 @@ NOTE: Do not install {DATAGRID} on the same node as {CENTRAL}.
...
+----
++
+with
++
+[source]
+----
+-Xms256m -Xmx2048m -XX:MetaspaceSize=256M
+----
++
+. Open `__JDG_HOME__/server/conf/infinispan.xml` and replace
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc]: As above.

**Comment by adrielparedes** [comment]
Documentation looks fine to me, LGTM!

**Comment by emmurphy1** [comment]
Approved

**Pull request merged**